app/src/i18n/index.ts

```diff
@@ -32,3 +33,10 @@ export async function useTranslation(
     i18n: i18nextInstance,
   };
 }
+
+export const getTranslationFromDict = (
+  translations: Record<string, string> | string | undefined,
+): string | undefined => {
+  const lang = i18next.language;
```

Comment:
I think this means that it won't update on the fly when the language changes? So maybe better to `useEffect` that registers this callback: https://www.i18next.com/overview/api#onlanguagechanged
And then unregisters it in the returned function.
If it automatically re-renders when you select a different language in the language dropdown in the navigation bar then this is fine :sparkles:

Reply:
I just checked, in automatically re-renders